lib/templates.php

```diff
@@ -232,3 +264,16 @@ function filter_rest_wp_template_query( $args, $request ) {
 	return $args;
 }
 add_filter( 'rest_wp_template_query', 'filter_rest_wp_template_query', 99, 2 );
+
+/**
+ * Filter the default template types definitions to temporarily remove 'embed',
+ * because it is not a regular template type.
+ *
+ * @param array $template_type_definitions The array of template types definitions.
+ * @return array Filtered $template_type_definitions.
+ */
+function gutenberg_filter_default_template_types_definitions( $template_type_definitions ) {
+	unset( $template_type_definitions['embed'] );
```

Comment:
Is there a reason why have to include `embed` in the type definitions in the first place?

Reply:
It was included in the original list here:
https://github.com/WordPress/gutenberg/blob/40a5502d62534cd64a623938754778501264736c/lib/template-loader.php#L25
And then removed manually every time it's about to be used (try searching for `// Skip 'embed' for now` in the codebase).

While I could have just remove it from the list, I chose to include it in order to showcase the filter, which could be useful to see how it's used.